WorkloadManagementSystem/PilotAgent/pilotCommands.py

```diff
@@ -506,35 +511,35 @@ def execute( self ):
 
     maxRAM = self.pp.queueParameters.get('MaxRAM', maxRAM)
     if maxRAM:
-      self.cfg.append( '-o "/Resources/Computing/CEDefaults/MaxRAM=%d"' % maxRAM )
+      self.cfg.append('-o "/Resources/Computing/CEDefaults/MaxRAM=%d"' % maxRAM)
```

Comment:
I guess that gives a guaranteed merge conflict with #3623

Reply:
It's not impossible. This one is anyway for v6r20, I will take care of solving the conflicts if present.